internal/proto/vanguard/test/v1/library.proto

```diff
@@ -173,6 +180,12 @@ message CreateShelfRequest {
   Shelf shelf = 1;
 }
 
+message ListShelvesRequest {}
+
+message ListShelvesResponse {
+  repeated Shelf shelves = 1;
+}
+
```

Comment:
Small addition to make the request look more like a real list request. This proto file acts as documentation for building a REST style API.

```suggestion
message ListShelvesRequest {
  // The maximum number of items to return.
  int32 page_size = 1;

  // The next_page_token value returned from a previous List request, if any.
  string page_token = 2;
}

message ListShelvesResponse {
  // The list of shelves.
  repeated Shelf shelves = 1;
  // Token to retrieve the next page of results, or empty if there are no
  // more results in the list.
  string next_page_token = 2;
}
```